distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -656,6 +658,113 @@ type Column
                 rs = s.iif true_val false_val storage_type
                 Column.Value (Java_Column.new new_name rs)
 
+    ## Round the values in a numeric column to a specified number of decimal
+       places.
+
+       For integers, rounding to 0 or more decimal places simply returns the
+       argument. For negative decimal places, see below.
+
+       By default, rounding uses "asymmetric round-half-up", also known as
+       "round towards positive infinity." If use_bankers=True, then it uses
+       "round-half-even", also known as "banker's rounding".
+
+       If the column is of type `Decimal` and `decimal_places` > 0, `round`
+       returns a column of `Decimal`; otherwise, it returns a column of
+       `Integer`.
```

Comment:
```suggestion
       If the column is of type `Float` and `decimal_places` > 0, `round`
       returns a column of `Float`; otherwise, it returns a column of
       `Integer`.
```

Unfortunately with `Value_Type` with have a bit of inconsistency between it and the Enso types.

<table>
<tr>
 <th> Value Type
 <th> Enso Type
<tr>
 <td>`Value_Type.Integer`
 <td>`Integer`
<tr>
 <td>`Value_Type.Float`
 <td>`Decimal`
<tr>
 <td>`Value_Type.Decimal`
 <td>(currently not supported) `BigDecimal`
</table>

The names for value type were chosen to be close to the SQL type names. Maybe we should rename Enso `Decimal` into `Float`?? (cc @jdunkerley) It is indeed a bit confusing to have this inconsistency. Also indeed, `Decimal` often suggests precise decimals, where we have IEEE-754 floating points.

Anyway, regardless of the possible naming discussion (which is rather out of scope of this PR), when talking about _column types_, please remember to use the Value Type nomenclature.

Reply:
I agree `Decimal` implies a precise decimal - that one is probably a wider discussion. One to raise with @wdanilo I guess.